model/resolver/LDFResource.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+class LDFResource implements RemoteResource
+{
+    private $uri;
+    private $ldfEndpoint;
+
+    public function __construct(string $uri, string $ldfEndpoint) {
+        $this->uri = $uri;
+        $this->ldfEndpoint = $ldfEndpoint;
+    }
+    
+    public function resolve(int $timeout) : ?EasyRdf\Resource {
+        try {
+            // change the timeout setting for external requests
+            $httpclient = EasyRdf\Http::getDefaultHttpClient();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'EasyRdf\Http' in method 'resolve'.](https://app.codacy.com/manual/NatLibFi/Skosmos/pullRequest?prid=4951447)